EasyPost.Tests/ServicesTests/AddressServiceTest.cs

```diff
@@ -40,9 +40,8 @@ public async Task TestCreateVerify()
             UseVCR("create_verify");
 
             Dictionary<string, object> addressData = Fixtures.IncorrectAddress;
-            addressData.Add("verify", true);
 
-            Address address = await Client.Address.Create(addressData);
+            Address address = await Client.Address.Create(addressData, true);
```

Comment:
Please rollback these changes. As with other items, we were going to focus on updated params at a later date across all libs. Let's not do that now

Reply:
Resolved